DataversesIT.testMoveDataverse, DataversesIT.testImportDDI, FilesIT.testAccessFacet (again)

Even after this PR was merged...

- #9855

... we're still seeing the same tests fail at https://jenkins.dataverse.org/job/IQSS-dataverse-develop/1462/consoleFull

"[INFO] Results:", "[INFO] ", "[ERROR] Failures: ", "[ERROR]   DataversesIT.testImportDDI:537 1 expectation failed.", "JSON path data.total_count doesn't match.", "Expected: is <1>", "  Actual: null", "", "[ERROR]   DataversesIT.testMoveDataverse:381 1 expectation failed.", "JSON path data.total_count doesn't match.", "Expected: <1>", "  Actual: null", "", "[ERROR]   FilesIT.testAccessFacet:1124 1 expectation failed.", "JSON path data.total_count doesn't match.", "Expected: <1>", "  Actual: null", "", "[INFO] ", "[ERROR] Tests run: 210, Failures: 3, Errors: 0, Skipped: 8", "[INFO] ", "[INFO]

The same tests mentioned in the issue that PR was supposed to fix, that is:

- #9848

--- a/src/test/java/edu/harvard/iq/dataverse/api/AdminIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/AdminIT.java
@@ -734,6 +734,13 @@ public class AdminIT {
                 .statusCode(OK.getStatusCode());
     }
 
+    /**
+     * Disabled because once there are new fields in the database that Solr
+     * doesn't know about, dataset creation could be prevented, or at least
+     * subsequent search operations could fail because the dataset can't be
+     * indexed.
+     */
+    @Disabled
     @Test
     public void testLoadMetadataBlock_NoErrorPath() {
         Response createUser = UtilIT.createRandomUser();
@@ -778,6 +785,13 @@ public class AdminIT {
         assertEquals(244, (int) statistics.get("Controlled Vocabulary"));
     }
 
+    /**
+     * Disabled because once there are new fields in the database that Solr
+     * doesn't know about, dataset creation could be prevented, or at least
+     * subsequent search operations could fail because the dataset can't be
+     * indexed.
+     */
+    @Disabled
     @Test
     public void testLoadMetadataBlock_ErrorHandling() {
         Response createUser = UtilIT.createRandomUser();

--- a/src/test/java/edu/harvard/iq/dataverse/api/MoveIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/MoveIT.java
@@ -300,7 +300,7 @@ public class MoveIT {
                 .statusCode(OK.getStatusCode())
                 .body("feed.entry[0].id", CoreMatchers.endsWith(datasetPid));
 
-        UtilIT.sleepForReindex(datasetPid, superuserApiToken, 10);
+        UtilIT.sleepForReindex(datasetPid, superuserApiToken, 20);
         Response getLinksAfter = UtilIT.getDatasetLinks(datasetPid, superuserApiToken);
         getLinksAfter.prettyPrint();
         getLinksAfter.then().assertThat()

--- a/src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/UtilIT.java
@@ -563,7 +563,14 @@ public class UtilIT {
                 .post("/api/datasets/:persistentId/modifyRegistrationMetadata/?persistentId=" + persistentId);
         return response;
     }
-    
+
+    /**
+     * Deprecated because once there are new fields in the database that Solr
+     * doesn't know about, dataset creation could be prevented, or at least
+     * subsequent search operations could fail because the dataset can't be
+     * indexed.
+     */
+    @Deprecated    
     static Response loadMetadataBlock(String apiToken, byte[] body) {
         return given()
           .header(API_TOKEN_HTTP_HEADER, apiToken)

